lib/LANraragi/Plugin/Metadata/Fakku.pm

```diff
@@ -86,8 +93,26 @@ my $fakku_host = "https://www.fakku.net";
 # Uses the website's search to find a gallery and returns its gallery ID.
 sub search_for_fakku_url {
 
-    my $title  = $_[0];
-    my $logger = get_logger( "FAKKU", "plugins" );
+    my ($title) = @_;
+
+    my $dom = get_search_result_dom_by_title($title);
```

Comment:
Good: I agree to remove it :smile: 
Many others likes to see the required parameters in the subs' names because they don't have code completion and so I've got used to it, but I don't like it very much due to the increased length of the names...

Reply:
Yeah, this is one of the reasons I tend to put the expected arguments as a quick comment on top of the subs.  
I'd like to eventually move to sub signatures which would completely solve this, but I can't bother putting the experimental flag everywhere either so I'm just waiting for Perl 7. 🙃